deployment/terraform/dns.tf

```diff
@@ -0,0 +1,48 @@
+#
+# Private DNS resources
+#
+resource "aws_route53_zone" "internal" {
+  name       = "${var.r53_private_hosted_zone}"
+  vpc_id     = "${module.vpc.id}"
+  vpc_region = "${var.aws_region}"
+
+  tags {
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_route53_record" "database" {
+  zone_id = "${aws_route53_zone.internal.zone_id}"
+  name    = "database.service.${var.r53_private_hosted_zone}"
+  type    = "CNAME"
+  ttl     = "10"
+  records = ["${module.database.hostname}"]
+}
+
+#
+# Public DNS resources
+#
+resource "aws_route53_zone" "external" {
+  name = "${var.r53_public_hosted_zone}"
```

Comment:
Are we completely in charge of the DNS for this project? If we aren't, it may give us some more flexibility to manage hosted zones outside of terraform, and use a `r53_hosted_zone_id`, like we do in azavea/fund-equity-map#104